src/common/paddle/pricing.ts

```diff
@@ -160,6 +166,34 @@ export const getPricingMetadata = async (
   }
 };
 
+export const getPricingMetadataByPriceIds = async (
+  ctx: AuthContext,
+  ids: string[],
+): Promise<Record<string, BasePricingMetadata>> => {
+  // Because we disable escaping, we need to ensure that the ids are valid
+  // strings, just to be safe.
+  const pricingIds = z.array(z.string()).safeParse(ids);
+  if (pricingIds.error) {
+    return {};
+  }
+
+  const items = await ctx.con
+    .createQueryBuilder()
+    .select('item')
+    .from(ExperimentVariant, 'ev')
+    .addFrom('jsonb_array_elements(ev.value::jsonb)', 'item')
+    .disableEscaping()
```

Comment:
Have to disable escaping, otherwise it wraps `"` around the `jsonb_array_elements(ev.value::jsonb)` which is not valid psql :triggered:

Alternative is using a raw query